arroyo-console/src/components/WelcomeModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from 'react';
+import {
+  Button,
+  Link,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react';
+import { TourContext, TourSteps } from '../tour';
+
+const WelcomeModal: React.FC = () => {
+  const { tourStep, setTourStep, disableTour } = useContext(TourContext);
+
+  return (
+    <Modal
+      isOpen={tourStep == TourSteps.WelcomeModal}
+      onClose={() => {}}
+      isCentered
+      size={'xl'}
+      variant={'tour'}
+    >
+      <ModalOverlay />
+      <ModalContent padding={2}>
+        <ModalCloseButton onClick={disableTour} />
+        <ModalBody display={'flex'} gap={3} flexDirection={'column'} alignItems={'center'}>
+          <img width="320" src="/logo.svg" />
```

Comment:
I'd make the logo much smaller, I think it distracts too much from the content

Reply:
Done